pkg/aws/ec2.go

```diff
@@ -48,7 +52,7 @@ type AssertEC2TagValueInput struct {
 	TagName string
 	// The value of the tag to assert.
 	Value string
-	// The Instance ID that the tag mustbe set on.
+	// The Instance ID that the tag must be set on.
```

Comment:
☝️ Nonblocking nit: I confess that this language trips me up a bit; I'm not quite sure what it means without squinting a bit more at the context and code.

Reply:
I updated the description a bit, let me know if that makes sense?